tox.ini

```diff
@@ -1,14 +1,12 @@
 [tox]
-envlist = py35-django{111,20,21,22}-drf{39,310,latest}
+envlist = py{35,38}-django22-drf{39,310,latest}, py38-django30
```

Comment:
Won't we still want to test multiple DRF versions under Django 3.0?

Reply:
There isn't any released version of DRF which supports Django 3.0. Github's master branch of DRF has support for Django 3.0. Should we test django 3.0 against that?

Reply:
Or just use `latest`, and we'll see if they've released a new version with Django 3.0 support by the time we get around to actually testing it.  We're not actually adding Django 3.0 test configurations in Travis yet.

Reply:
Added drf latest with django 3.0